src/main/java/seedu/exchangecoursemapper/command/ListSchoolCommand.java

```diff
@@ -8,24 +10,36 @@
 import java.io.FileReader;
 import java.io.IOException;
 import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
```

Comment:
I like that you avoided the use of wildcard imports here, by listing out the methods we will be using from the package!